### DISCUSSION TITLE
A GUI for both neovim and terminal

### DISCUSSION THREAD

**Discussion by A7R7** [neovide/neovide#1875] [General]
Hi, Thank you for your great works! 
In neovim you can open a terminal, and the terminal works seamlessly in neovim. Therefore I'm wondering, is it possible to do a bit more development on neovide to make it not only a GUI for neovim, but also a GUI for a terminal? I think it would be an amazing work!

**Comment by Mange** [comment]
Maybe one could bind `C-z` to open a Neovim terminal in a new tab or something…

Then I only need to retrain not writing `fg` when I'm done in the terminal and instead close it. :thinking:

**Comment by PlexSheep** [comment]
I mean, the option to just use `:terminal` exists. That can be expanded upon with neovim plugins, like ToggleTerm.
